integration-tests/src/tests/client/flat_storage.rs

```diff
@@ -74,7 +68,7 @@ fn test_flat_storage_iter() {
         }
 
         if shard_id == s0 {
-            let expected = 2 + protocol_version_modifier;
+            let expected = 2 + bandwidth_scheduler_modifier;
```

Comment:
nit: I guess we can just hardcode the `expected` value to be 3 here. We can push the comment from below to above this line.